packages/mui-material-next/src/SnackbarContent/SnackbarContent.tsx

```diff
@@ -3,13 +3,13 @@ import * as React from 'react';
 import PropTypes from 'prop-types';
 import clsx from 'clsx';
 import { unstable_composeClasses as composeClasses } from '@mui/base/composeClasses';
-import { emphasize } from '@mui/system';
-import styled from '@mui/material/styles/styled';
-import useThemeProps from '@mui/material/styles/useThemeProps';
 import Paper from '@mui/material/Paper';
```

Comment:
This component depends on `Paper`, I wonder if this will be a blocker for implementing the MD3 design 👀